solutions/go/01-a01/code/app/main.go

```
@@ -0,0 +1,76 @@
+			var err error
+			fenStr := request["fen"].(string)
+			fen, err = chess.FEN(fenStr)
+			if err != nil {
+				fenStr += " 0 1"
+				fen, err = chess.FEN(fenStr)
+				if err != nil {
+					http.Error(w, err.Error(), http.StatusBadRequest)
+					return
+				}
+			}
+			fmt.Println(fen)
```

golangci-lint (1.64.8)

printf: fmt.Println arg fen is a func value, not called

(govet)